apps/explorer/src/components/transaction-card/TxCardUtils.tsx

```diff
@@ -71,22 +62,26 @@ export function SuiAmount({
     return <span className={styles.suiamount}>--</span>;
 }
 
-export function TxAddresses({ content }: { content: LinkObj[] }) {
+function TxTableHeader({ label }: { label: string }) {
+    return <div className="pl-3">{label}</div>;
+}
+
+function TxTableCol({
+    isHighlightedOnHover,
```

Comment:
As an aside I really hate how our table expects hover on rows. We probably want to either change the design, or rewrite all of this at some point not using tables so that we can implement the design in a sane way.

Reply:
I think this is due to the table expects data in col, but hovering on row. If we update to data by row instead of col, it would solve this issue, but we will run into other issues with proper styling going that way also :/ 
Let me know if you'd like this part to be changed at the moment though.